packages/components/filters/filter-menu/src/filter-menu.tsx

```diff
@@ -5,7 +5,7 @@ import { Header } from './header';
 import { TriggerButton } from './trigger-button';
 
 export type TAppliedFilterValue = {
-  key?: string;
+  value: string;
```

Comment:
the idea being that the `value` is a string that can be used as a key, and the `label` is a `ReactNode` that renders the applied value with the desired format inside the `Chip` in the `TriggerButton`